### PR TITLE
CLOUDP-172682 Clarify when APIKeyRole and APIKeyDescription are required when creating organizations

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Clarify when APIKeyRole and APIKeyDescription are required when creating organizations

_Jira ticket:_ CLOUDP-172682

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by gssbzn — APPROVED** [review]
LGTM

**Review by blva — APPROVED** [review]

**Review by sarahsimpers — APPROVED** [review]
LGTM

File: internal/cli/iam/organizations/create_atlas.go [hunk 1]
```diff
@@ -1,162 +0,0 @@
-// Copyright 2023 MongoDB Inc
```
Comment (fmenezes) [inline internal/cli/iam/organizations/create_atlas.go:1]: unused file

File: internal/cli/iam/organizations/create_atlas_test.go [hunk 2]
```diff
@@ -1,65 +0,0 @@
-// Copyright 2023 MongoDB Inc
```
Comment (fmenezes) [inline internal/cli/iam/organizations/create_atlas_test.go:1]: unused file

**Pull request merged**